Makefile

```diff
@@ -96,8 +95,18 @@ generate: $(KUSTOMIZE) go.generate manifests.annotate
 # policy is necessary to be able to redeploy helm chart after it is deleted
 # since CRDs with "crd-install" hooks will not be deleted with "helm delete" and
 # cause next "helm install" to fail.
-CRD_DIR = cluster/charts/crossplane/templates/crds
+# "helm.sh/hook: crd-install" deprecated in helm3 and crds with this annotation
```

Comment:
Could this comment be cleaned up and explained just a bit more?  e.g.,

```make
# "helm.sh/hook: crd-install" was deprecated in helm3 and CRDs with this annotation are
# skipped. This results in the StackInstall and ClusterStackInstall CRDs not being
# installed in helm3 when they have that annotation.
# As a workaround, we first copy those CRDs under <chart>/crds directory which
# was introduced with helm3 and ignored in helm2, then afterwards apply the annotation to
# those CRDs under <chart>/templates/crds for helm2.
```

Reply:
Done! Thanks for the suggested text.